src/chroot.c

```diff
@@ -75,6 +75,17 @@ wrapper(chroot, int, (const char * path))
         return -1;
     }
 
+    if (strcmp(path, ".") == 0) {
```

Comment:
Is there a helper that determines if the `path` is outside `fakechroot_base`? It is working great for `dnf` for now.

_Note_: I will do more testing to understand how `chroot()` is working.